src/Json_encode.mli

```diff
@@ -18,6 +18,14 @@ external int : int -> Js.Json.t = "%identity"
 external boolean : Js.boolean -> Js.Json.t = "%identity" 
 (** [boolean b] makes a JSON boolean of the [Js.boolean] [b] *)
 
+val bool : bool -> Js.Json.t
+
+val optional : 'a encoder -> 'a option -> Js.Json.t
```

Comment:
Perhaps this should be called `nullable` instead, since it's not quite obvious that `None` will result in a `null`, rather than `undefined`